### PR TITLE
PDE-5004 - Add page about request template in auth

### PR DESCRIPTION
**Pull request by standielpls**

### CODE REVIEW

**Comment by justin-lee-zapier** [comment]
When you're ready for @team-platform-support to review this, please head over to this [Slack thread](https://zapier.slack.com/archives/C02PCUNBUG7/p1715360777565129) and give us a `:review-pr:` emoji.

File: docs/_build/authentication/requesttemplate.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+title: Add authentication fields to Request Template
+order: 10
+layout: post-toc
+
+---
+
+# Add authentication fields to Request Template
+
+The Request Template is a request editor that lets users set static values that apply to all requests made by this integration. Users can configure the URL params, HTTP headers and request body. This is the perfect place to 
+set authentication fields.
```
Comment (marinahand) [inline docs/_build/authentication/requesttemplate.md:11]: run on line, should this be in the same paragraph?

File: docs/_build/authentication/requesttemplate.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+![](https://cdn.zappy.app/987abd7e65935ecb581fd356b45fb4bc.png)
+
+Adding fields to the Authentication Fields also writes the fields to the Request Template.
```
Comment (marinahand) [inline docs/_build/authentication/requesttemplate.md:15]: suggestion:
"Adding input fields in the Authentication Fields section.."

File: docs/_build/authentication/requesttemplate.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+The Request Template is a request editor that lets users set static values that apply to all requests made by this integration. Users can configure the URL params, HTTP headers and request body. This is the perfect place to 
+set authentication fields.
+
+![](https://cdn.zappy.app/987abd7e65935ecb581fd356b45fb4bc.png)
+
+Adding fields to the Authentication Fields also writes the fields to the Request Template. 
+![](https://cdn.zappy.app/8cfafbc546e1693fa1e517ccc14a96ab.png)
+
+By adding fields to the Request Template, they will also have it show up in the Trigger and Actions request builder.
```
Comment (marinahand) [inline docs/_build/authentication/requesttemplate.md:18]: suggestion:
By adding fields to the Request Template, they will also show up in the request builder of Triggers and Actions.

File: docs/_build/authentication/requesttemplate.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+The fields will be disabled with help text, and will prompt the user to go to the Request Template under Advanced Settings if they want to make changes to it.
```
Comment (marinahand) [inline docs/_build/authentication/requesttemplate.md:19]: The applicable fields for authorization in triggers and actions under `Options` would then be disabled with help text. You'll need to access the Request Template under Advanced Settings to make changes to them.

**Review by marinahand — COMMENTED** [review]
Thanks for the new doc! A couple of edits to match the style of the docs :)

**Review by marinahand — APPROVED** [review]

**Pull request merged**